ui/Face.qml

```diff
@@ -0,0 +1,133 @@
+import QtQuick.Layouts 1.4
```

Comment:
should there be a Mark II version of this file that uses the new design best practices?

Reply:
Maybe - though the current version looks good to me, and I don't believe there's a Mark II grid design in Figma. So I'm suggesting that's out of scope for this PR.